vector/build.gradle

```diff
@@ -157,10 +157,13 @@ android {
         buildConfigField "Boolean", "ENABLE_CROSS_SIGNING", "${enableCrossSigning}"
         resValue "bool", "enable_cross_signing", "${enableCrossSigning}"
 
-        // Tchap: Spaces feature is hidden
+        // Tchap: Spaces feature is hidden, show all rooms in home
```

Comment:
Same comment: Show all in the room. The spaces are hidden.

Reply:
Also updated, I think the documentation is quite correct here